docs/develop/node/archival/run-archival-node-without-nearup.md

```diff
@@ -15,7 +15,7 @@ Running an archival node is very similar to running a [validator node](/docs/dev
 
 ## Prerequisites {#prerequisites}
 
-- [Rust](https://www.rust-lang.org/). If not already installed, please [follow these instructions](https://docs.near.org/docs/tutorials/contracts/intro-to-rust#3-step-rust-installation).
+- [Rust](https://www.rust-lang.org/). If not already installed, please [follow these instructions](https://docs.near.org/docs/tutorials/contracts/intro-to-rust#3-step-rust-installation). If already installed, please note the compiler version in [rust-toolchain](https://github.com/near/nearcore/blob/master/rust-toolchain.toml)
```

Comment:
I'd rephrase it along the following lines

```
Latest stable version of Rust. We recommend using https://rustup.rs to manage installation
```

That is:

* latest stable, rather than `rust-toolchain.toml`. We generally support latest stable and this additionally gives the info "you probably won't be able to use outdated rust from your linux distro"
* don't link to *our* docs on how to install Rust, link to the Rust docs instead.